package.json

```diff
@@ -59,19 +62,22 @@
     "expo-linear-gradient": "~11.3.0",
     "expo-localization": "~13.0.0",
     "expo-random": "~12.2.0",
-    "expo-splash-screen": "^0.17.5",
+    "expo-splash-screen": "~0.15.1",
     "expo-status-bar": "~1.3.0",
     "expo-updates": "~0.13.4",
     "fast-text-encoding": "^1.0.3",
+    "fbjs": "^3.0.4",
     "i18n-js": "^3.8.0",
     "jose": "^4.10.0",
     "jwt-decode": "^3.1.2",
     "lodash.memoize": "^4.1.2",
     "metro-config": "^0.73.3",
     "process": "^0.11.0",
+    "prop-types": "^15.8.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-native": "0.68.5",
+    "react-is": "^18.2.0",
+    "react-native": "0.68.2",
```

Comment:
This version of react-native has issues. This was the big skrewup by the RN developers. That is why 0.68.5 was released